### PR TITLE
story-009 :  Updates for Dsahboard Details

### PR DESCRIPTION
**Pull request by nurub52**
Add Support to Update Dashboard Details from API
Add Contributor
Added fetature to broadcas message on dashboard

Story:  https://trello.com/c/CfMZeDb1/15-phase-1-story-009-dashboard-details

### CODE REVIEW

File: client/src/index.tsx [hunk 1]
```diff
@@ -19,4 +19,4 @@ ReactDOM.render(
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: http://bit.ly/CRA-PWA
-serviceWorker.register();
+//serviceWorker.register();
```
Comment (hussainweb) [inline client/src/index.tsx]: @nurub52, if you want to not have a service worker for now, it is better to just revert it to `serviceWorker.unregister()`. That makes sure that the previous SWs are cleared.

**Review by hussainweb — COMMENTED** [review]
I see some files got reverted from previous fixes and some of the changes are not really relevant. That's not a blocker. I am still working on the clean-up and I can just repeat them in the next PR. For now, please see my comment about the service worker.

**Comment by nurub52** [comment]
updated and pushed code as mentioned.

File: client/src/index.tsx (hunk 1 above)
Comment (nurub52) [reply client/src/index.tsx]: updated as commented

**Pull request merged**